.goreleaser.yml

```diff
@@ -50,4 +50,4 @@ release:
     name: doctl
 
 changelog:
-  skip: true
+  skip: false
```

Comment:
Will this switch the changelog on GitHub from the one we generate from https://github.com/digitalocean/github-changelog-generator to one generated by GoReleaser??

Reply:
Yes. That's what I wanted to try.